linux_os/guide/services/fapolicyd/package_fapolicyd_installed/rule.yml

```diff
@@ -25,14 +25,15 @@ references:
     stigid@ol8: OL08-00-040135
     stigid@rhel8: RHEL-08-040135
 
-ocil_clause: 'the package is not installed'
+ocil_clause: 'the fapolicyd package is not installed'
 
 ocil: '{{{ ocil_package(package="fapolicyd") }}}'
 
 fixtext: |-
-    {{{ describe_package_install("fapolicyd") }}}
+    {{{ fixtext_package_installed("fapolicyd") | indent(4) }}}
 
-srg_requirement: '{{{ srg_requirement_package_installed("fapolicyd") }}}'
+srg_requirement: |-
+    The {{{ full_name }}} fapolicy module must be installed.
```

Comment:
Should be called here by `fapolicy module` or `fapolicyd package` to be aligned to the ocil_clause and fixtext?